frontend/app-development/layout/SettingsModalButton/SettingsModal/SettingsModal.tsx

```diff
@@ -131,11 +131,7 @@ export const SettingsModal = ({
     >
       <div className={classes.modalContent}>
         <div className={classes.leftNavWrapper}>
-          <LeftNavigationBar
-            tabs={leftNavigationTabs}
-            className={classes.leftNavigationBar}
-            selectedTab={currentTab}
-          />
+          <LeftNavigationBar tabs={leftNavigationTabs} selectedTab={currentTab} />
```

Comment:
Is the className supposed to be removed?

Reply:
It did not exist so I guess it is safe to remove it 🫣